hud-data/bania-hud.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "Bania Hud",
+  "": "a hud i made lolololol",
+  "author": "smh2211",
+  "contributors": [
+    "Hypnotize",
+    "Chippy",
+    "Village Green Preserver",
+    "weareallimaginary"
+  ],
+  "releaseDate": "2024-05-20",
+  "resources": [
+    "huds.tf discord",
+    "fontforge",
+    "paint.net",
+    "notepad++"
+  ],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "materials"
+  ],
+  "tags": [
+    "minimal",
+    "box",
+    "black and white",
+    ""
+  ],
+  "repo": "https://github.com/HUBM3N/Bania-Hud",
+  "hash": "6b5e53b451b02348c1bdbd6d00b00537eaeb8738",
+  "prerelease": false,
+  "social": {
+    "steam_profile": "76561198992606708",
+    "discord": "cowshit",
+    "album": "https://imgur.com/a/ULn1arh"
+  },
+  "traits": {
+    "os": [
+      "windows"
+    ],
+    "gamemode": [
+      "tournament",
+      "comp",
+      "arena",
+      "ad",
+      "ctf",
+      "cp",
+      "koth",
+      "pl",
+      "plr",
+      "pd"
+    ],
+    "res": [
+      "4/3",
+      "16/9",
+      "16/10"
+    ],
+    "position": [
+      "center"
+    ],
+    "lang": [
+      "English"
```

```suggestion
      "en"
```
you use the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) codes for these (presumably at least, even though tf2 has a few other ones used i think)

just change it to en